backend/hdf5/BlockHDF5.cpp

```diff
@@ -290,15 +292,16 @@ shared_ptr<ITag> BlockHDF5::createTag(const std::string &name, const std::string
 shared_ptr<IDataArray> BlockHDF5::createDataArray(const std::string &name,
                                                   const std::string &type,
                                                   nix::DataType data_type,
-                                                  const NDSize &shape) {
+                                                  const NDSize &shape,
+                                                  const Compression &compression) {
     string id = util::createId();
     boost::optional<H5Group> g = data_array_group(true);
 
     H5Group group = g->openGroup(name, true);
     auto da = make_shared<DataArrayHDF5>(file(), block(), group, id, type, name);
 
-    // now create the actual H5::DataSet
-    da->createData(data_type, shape);
+    // now create the actual H5::Data Set
```

Comment:
Nitpick: `H5::DataSet` was correct ;)

Reply:
args, struggling with the emacs settings, in particular the indentation... Thought I found them all before pushing, apparently not.

Reply:
I think I fucked the emacs stuff up for us, fixating...